sql-jdbc/src/TableauConnector/opensearch_sql_jdbc/dialect.tdd

```diff
@@ -65,62 +65,62 @@
         </function>
 
         <function group='operator' name='==' return-type='bool'>
-            <formula>(CAST(%1 as TIMESTAMP) = CAST(%2 as TIMESTAMP))</formula>
+            <formula>(CAST(DATE_FORMAT(%1, &apos;%Y-%m-%d %H:%i:%s&apos;) AS TIMESTAMP) = CAST(DATE_FORMAT(%2, &apos;%Y-%m-%d %H:%i:%s&apos;) as TIMESTAMP))</formula>
```

Comment:
Is an alternative here to just use the TIMESTAMP function directly instead of the combo CAST/DATE_FORMAT?